recipes/imgviz/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "imgviz" %}
+{% set version = "0.10.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5bc8167ab284c853e55a429f5529b90faba816e7247df18fb918610f29e0329a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - matplotlib-base
+    - numpy
+    - pillow >=5.3.0
+    # contained in [all]
+    - scikit-image
+    - scikit-learn
+
+test:
+  imports:
+    - imgviz
+
+about:
+  home: https://github.com/wkentaro/imgviz
+  # Actually, the license is unkown, I simply took one from a similar project
+  # from the same author
+  # https://github.com/wkentaro/labelme
+  # Upstream asked for clarification:
+  # https://github.com/wkentaro/imgviz/issues/11
+  license: GPL-3.0
```

Comment:
The license type listed on PyPI is MIT, https://pypi.org/project/imgviz/, and the setup file has listed it as MIT as well, https://github.com/wkentaro/imgviz/blob/master/setup.py.

Reply:
Thanks! I updated it as much as I could. Upstream should definitely clarify.

Reply:
Thanks Mark! Let's give the developer some time to reply then.